app/functions.py

```diff
@@ -64,7 +64,7 @@ def processing_job(encryptedRecord, redisHost, redisPort):
 
     # write out the results
     results = Results(LOGGER, redisHost, redisPort)
-    results.write_result(job.id, str(record))
+    results.write_result(job.id, record)
```

Comment:
Should we encrypt the results?

Reply:
The write_result method encrypts the results.